app/controllers/commontator/comments_controller.rb

```diff
@@ -198,8 +198,13 @@ def update_unread_status
       return unless medium.released.in?(['all', 'users', 'subscribers'])
 
       relevant_users = medium.teachable.media_scope.users
-      relevant_users.where(unread_comments: false)
+      relevant_users.where.not(id: current_user.id)
+                    .where(unread_comments: false)
                     .update_all(unread_comments: true)
-      @update_icon = relevant_users.exists?(current_user.id)
+
+      # make sure that the generated comment is marked as read by the creator
+      # so it is not listed as a new comment in the comment index
+      Reader.find_or_create_by(user: current_user, thread: @commontator_thread)
+            .touch
```

Comment:
`touch()`: "Please note that no validation is performed" ~ from [here](https://apidock.com/rails/ActiveRecord/Persistence/touch).

Is there another method that _does_ perform validation? Or why do we skip it? Instead, couldn't we use `save()`, which by default performs validation?